doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,414 @@
+
+.. code-block:: text
+
+    subsection cfd-dem
+      set grad div                      = true
+      set void fraction time derivative = true
+      set drag force                    = true
+      set buoyancy force                = true
+      set shear force                   = true
+      set pressure force                = true
+      set drag model                    = rong
+      set coupling frequency            = 100
+      set grad-div length scale         = 0.005
+      set vans model                    = modelA
+    end
+
+
+We determine the drag model to be used for the calculation of particle-fluid forces. We enable buoyancy, drag, shear and pressure forces. For drag, we use the Rong model to determine the momentum transfer exchange coefficient. The VANS model we are solving is model A. Other possible option is model B.
+
+
+Non-linear Solver
+~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection non-linear solver
+      subsection fluid dynamics
+        set solver           = inexact_newton
+        set tolerance        = 1e-8
+        set max iterations   = 10
+        set verbosity        = verbose
+        set matrix tolerance = 0.75
+      end
+    end
+
+We use the ``inexact_newton`` solver as to avoid the reconstruction of the system matrix at each Newton iteration. For more information about the non-linear solver, please refer to the `Non Linear Solver Section <../../../parameters/cfd/non-linear_solver_control.html>`_
+
+Linear Solver
+~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection linear solver
+      subsection fluid dynamics
+        set method                                = gmres
+        set max iters                             = 5000
+        set relative residual                     = 1e-3
+        set minimum residual                      = 1e-10
+        set preconditioner                        = ilu
+        set ilu preconditioner fill               = 0
+        set ilu preconditioner absolute tolerance = 1e-12
+        set ilu preconditioner relative tolerance = 1
+        set verbosity                             = verbose
+        set max krylov vectors                    = 200
```

Comment:
```suggestion
        set max iters                             = 500
        set relative residual                     = 1e-3
        set minimum residual                      = 1e-10
        set preconditioner                        = ilu
        set ilu preconditioner fill               = 0
        set ilu preconditioner absolute tolerance = 1e-12
        set ilu preconditioner relative tolerance = 1
        set verbosity                             = verbose
        set max krylov vectors                    = 500
```

Do you need that many krylov vectors / linear solver iteration? Otherwise put the default value. This should be a simple problem to solve.